src/mmw/apps/home/urls.py

```diff
@@ -12,6 +12,7 @@
     url(r'^$', home_page, name='home_page'),
     url(r'^projects/$', projects, name='projects'),
     url(r'^project/$', project, name='project'),
+    url(r'^project/new/', project, name='project'),
```

Comment:
Is this just a fallback, since there is no actual functionality specific to `/project/new` in the back-end?